google-cloud-spanner-hibernate-samples/spring-data-jpa-sample/src/main/resources/application.properties

```diff
@@ -0,0 +1,24 @@
+# Application configuration to use Cloud Spanner with Spring Data JPA
+
+# Spanner connection URL.
+# - ${PROJECT_ID} Replace with your GCP project ID
+# - ${INSTANCE_ID} Replace with your Spanner instance ID
+# - ${DATABASE_NAME} Replace with your Spanner database name within your Spanner instance
+spring.datasource.url=jdbc:cloudspanner:/projects/${PROJECT_ID}/instances/${INSTANCE_ID}/databases/${DATABASE_NAME}
+
+# Specify the Spanner JDBC driver.
+spring.datasource.driver-class-name=com.google.cloud.spanner.jdbc.JdbcDriver
+
+# Specify the Spanner Hibernate dialect.
+spring.jpa.properties.hibernate.dialect=com.google.cloud.spanner.hibernate.SpannerDialect
+
+spring.jpa.hibernate.ddl-auto=update
+
+# Settings to enable batching statements for efficiency
+spring.jpa.properties.hibernate.jdbc.batch_size=100
+
+# You may display SQL statements and stats for debugging if needed.
+spring.jpa.properties.hibernate.generate_statistics=true
```

Comment:
I dunno what exactly is causing it by there is a lot of output like 
```
2019-10-28 11:57:33.263  INFO 151662 --- [nio-8080-exec-9] i.StatisticalLoggingSessionEventListener : Session Metrics {
    74572938 nanoseconds spent acquiring 1 JDBC connections;
    0 nanoseconds spent releasing 0 JDBC connections;
    290315 nanoseconds spent preparing 2 JDBC statements;
    128675013 nanoseconds spent executing 2 JDBC statements;
    0 nanoseconds spent executing 0 JDBC batches;
    0 nanoseconds spent performing 0 L2C puts;
    0 nanoseconds spent performing 0 L2C hits;
```
That we probably can disable.